courses/free-courses-pt_BR.md

```diff
@@ -22,6 +22,7 @@
 * [Jekyll](#jekyll)
 * [Kotlin](#kotlin)
 * [Kubernetes](#kubernetes)
+* [Laravel](#laravel)
```

Comment:
Laravel goes inside PHP 😉

Reply:
Okay. Thank you !!!

Reply:
If there are problems to merge as I can see, be sure that your for master/branch is synced with FPB master before start any task.

I see in your repo that is detached due to some PRs over yourself.

Maybe this helps you to solve it without delete repo and fork it again. https://github.com/davorpa/shell-utils/blob/cf6d2aeff574abe2fc9125e14071e62bfdb11f2d/scripts/git-refork-upstream-master

```bash
#git remote add upstream https://github.com/some_user/some_repo
git fetch origin
git fetch upstream
git checkout master
git reset --hard upstream/master
git push origin master --force
```